rules/sonic-mgmt-framework.mk

```diff
@@ -0,0 +1,7 @@
+# SONiC mgmt-framework package
+
+SONIC_MGMT-FRAMEWORK = sonic-mgmt-framework_1.0-01_amd64.deb
+$(SONIC_MGMT-FRAMEWORK)_SRC_PATH = $(SRC_PATH)/sonic-mgmt-framework
```

Comment:
Provide _DBG_DEPENDS

refer swss.mk & docker-orchagent.mk

Reply:
Thanks for the feedback. I have added debug targets now.